preprocess.py

```diff
@@ -109,14 +109,47 @@ def generate_stat_queries(samples):
     
     return query_list
 
+#function to generate info queries
+def generate_info_queries(samples):
+    verb_list = ["do", "know"]
+    cnt = 0
+    query_list = []
+    while cnt < samples:
+        randq = randint(0, 6)
+        if randq <= 2:
+            verb = choice(verb_list)
+            query = "what are you,info\n"
+            query1 = "what do you {},info\n".format(verb)
+            query2 = "what can you {},info\n".format(verb)
+            Qs = [query1, query2]
+            query_list.append(choice(Qs))
+        elif randq == 3:
+            query = "who are you,info\n"
+            query1 = "who made you,info\n"
+            Qs = [query, query1, query2]
```

Comment:
I'm guessing this is supposed to be `Qs = [query, query1]`?

Reply:
good catch! just pushed the fix